source/connect.txt

```diff
@@ -122,6 +122,52 @@ option for programmatic usage of ``mongosh``, like a :driver:`driver
    - To provision access to a MongoDB deployment, see :manual:`Database
      Users </core/security-users/>`.
 
+
+Connect with LDAP
+~~~~~~~~~~~~~~~~~
+
+When using LDAP for authorization, users connecting via
+``mongosh`` must:
+
+- Set :option:`--authenticationDatabase <mongosh --authenticationDatabase>` to ``$external``.
```

Comment:
This _should_ already be the implied default for the auth mechanisms listed here, so shouldn’t be necessary to set explicitly

Reply:
Got it, will remove this line from the edit.